### PR TITLE
Sync latest v3.0 ps2/rileylink_ios

### PR DESCRIPTION
**Pull request by darinkrauss**
Need to incorporate some of the latest work from `ps2/rileylink_ios` into the `NightscoutService` for pull synchronization.

### CODE REVIEW

**Review by darinkrauss — COMMENTED** [review]
@ps2 @nhamming Most of these changes are due to .strings file updates. AFAIK, the only files relevant to Tidepool Loop are contained in NightscoutUploadKit. I reviewed the three files changed in NightscoutUploadKit and would approve the PR if I could (but cannot because I created it). 

Also, I lightly reviewed all of the other code-related files and they looked fine, as well, from what I could tell.

File: NightscoutUploadKit/NightscoutProfile.swift [hunk 1]
```diff
@@ -114,7 +114,7 @@ public struct TemporaryScheduleOverride {
     let duration: TimeInterval
     let name: String?
 
-    public init(targetRange: ClosedRange<Double>?, insulinNeedsScaleFactor: Double?, symbol: String?, duration: TimeInterval, name: String?) {
+    public init(duration: TimeInterval, targetRange: ClosedRange<Double>?, insulinNeedsScaleFactor: Double?, symbol: String?, name: String?) {
```
Comment (nhamming) [inline NightscoutUploadKit/NightscoutProfile.swift:117]: [nit] Update the order of setting the properties to make the parameter order.

**Review by nhamming — APPROVED** [review]
Looked closely at the night scout files and LGTM. 

One question. If I understand you correctly, should we be relying on dependencies that aren't targeted for Tidepool Loop 1.0 (i.e., will rileylink_ios be included)?

**Comment by ps2** [comment]
@nhamming Tidepool Loop will not be using any of these frameworks; Tidepool has committed to not breaking DIY Loop in an incompatible way, so when Tidepool makes changes (like the services work), it also updates the DIY parts that would otherwise break.

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**